applications/web/Chart.yaml

```
@@ -9,4 +9,4 @@ keywords:
   - web
 name: web
 type: application
-version: 0.1.0
+version: 0.1.1
```

```suggestion
version: 0.2.0
```

We should increment the minor version since this is a backwards-compatible feature addition (this is the default behavior of the CI/CD agent as well).

---

gotcha!